### PR TITLE
Schema ui fixes

### PR DESCRIPTION
**Pull request by kpsherva**
Added:
testData dependant on the test files used for backend. It works like a fake integration test, if something changes in the schema on the backend (or frontend) the frontend tests will fail immediately (snapshots) and it should help us to spot the missing changes quicker.

### CODE REVIEW

File: invenio_app_ils/mappings/v6/documents/document-v1.0.0.json [hunk 1]
```diff
@@ -18,29 +46,55 @@
...
+                "curated_relation": {
+                  "type": "boolean"
+                },
+                "value": {
+                  "type": "keyword"
+                }
+              }
+            },
+            "alternative_names": {
+              "type": "keyword"
```
Comment (topless) [inline invenio_app_ils/mappings/v6/documents/document-v1.0.0.json:75]: There are a lot of "type": "keywords", i.e. `volume`, `note` etc, which I think they should be `text`.

File: invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentMetadata/DocumentMetadata.js [hunk 2]
```diff
@@ -317,7 +329,7 @@ export default class DocumentMetadata extends Component {
             <Grid.Column>
               <Container>
                 <Header as="h3">Abstract</Header>
-                <p>{document.metadata.abstracts}</p>
+                <p>{document.metadata.abstracts[0].value}</p>
```
Comment (topless) [inline invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentMetadata/DocumentMetadata.js:332]: We can iterate on the abstracts here and display each one in a `p` element.

**Comment by topless** [comment]
I have a more general comment for the schemas in general. It seems that the schemas start to grow wildly especially the document one. I have tried in the past to use `#definitions` but I didn't manage to make it work. Maybe we should consider breaking them down to smaller more reusable ones. 

Before that, an easy win we can do is to use a utility like [sort-json](https://github.com/richie5um/vscode-sort-json) for vscode, and sort alphabetically the properties.

File: invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentMetadata/DocumentMetadata.js (hunk 2 above)
Comment (ntarocco) [reply invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentMetadata/DocumentMetadata.js:332]: UX is gonna come especially when we have some real data :)

**Review by ntarocco — APPROVED** [review]

**Pull request merged**